woocommerce-gateway-ebanx/gateways/class-wc-ebanx-gateway.php

```diff
@@ -944,6 +944,15 @@ protected function save_user_meta_fields($order)
 	protected function process_response_error($request, $order)
 	{
 		$code = isset($request->status_code) ? $request->status_code : 'GENERAL';
+
+		if ('GENERAL' === $code
+			&& isset($request->payment->transaction_status)
+			&& $request->payment->transaction_status->code === 'NOK') {
+
+			$code = 'REFUSED-CC';
+			$status_message = $request->payment->transaction_status->description;
```

Comment:
This variable is overwritten in the next line of code. This value will never be used.